pkg/apis/hive/v1/clusterdeployment_types.go

```diff
@@ -431,6 +431,18 @@ type CertificateBundleStatus struct {
 	Generated bool `json:"generated"`
 }
 
+// RelocateStatus is the status of a cluster relocate
+type RelocateStatus string
```

Comment:
Noting the annotation it's used in would be useful context.

Reply:
I'm still not seeing the annotation it's used in mentioned here, something missed?

Reply:
Oh, I added it to `ClusterRelocateStatus` but not here. That brings to light that I have a `ClusterRelocateStatus` and a `RelocateStatus`. That is too confusing. I need to rename one or both of those.